lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
Note: Looks like the `compiled` part of the path is either `process.env.NODE_BINDINGS_COMPILED_DIR || "compiled"`

To simplify things we could remove that part by calling `require('bindings')({ compiled: '' })`, which would make our path `module_root/version/platform/arch/bindings`